package.json

```
@@ -43,7 +44,8 @@
     "eslint-config-prettier": "8.8.0",
     "eslint-plugin-jest": "27.2.2",
     "husky": "8.0.3",
-    "jest": "29.5.0",
+    "jest": "29.6.1",
+    "jest-junit": "^16.0.0",
```

```suggestion
    "jest-junit": "16.0.0",
```

Can we remove the `^`? The programme has asked us to ensure we have fixed versions of our dependencies!